src/main/java/taxi/controller/authentication/LoginController.java

```diff
@@ -0,0 +1,42 @@
+package taxi.controller.authentication;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.exception.AuthenticationException;
+import taxi.lib.Injector;
+import taxi.model.Driver;
+import taxi.service.AuthenticationService;
+
+@WebServlet(urlPatterns = "/login")
+public class LoginController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private static final AuthenticationService authenticationService
+            = (AuthenticationService) injector.getInstance(AuthenticationService.class);
```

Comment:
```suggestion
    private static final AuthenticationService authenticationService =
            (AuthenticationService) injector.getInstance(AuthenticationService.class);
```
i think like this is right